libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergContainerFragment.java

```diff
@@ -85,7 +88,8 @@ public void onCreate(Bundle savedInstanceState) {
                 postType,
                 isNewPost,
                 localeString,
-                translations);
+                translations,
+                getContext().getResources().getColor(R.color.background_color));
```

Comment:
We have defined background color in our styles.xml for both light and dark mode, so I think we can use an existing color attribute here, like this:
`ContextExtensionsKt.getColorFromAttribute(getContext(), R.attr.backgroundColor)`

Reply:
ContextExtensionsKt isn't available in this class. Not sure if we want to include it?